spec/killbill/helpers/payment_plugin_spec.rb

```diff
@@ -308,6 +308,17 @@
       source.first_name.should == 'John'
       source.last_name.should == 'Doe'
     end
+
+    it "should support search_payments" do
+      response = ::Killbill::Test::TestResponse.create(
+        :api_call => :purchase, :kb_payment_id => @kb_payment_id,
+        :kb_account_id => @kb_account_id, :kb_tenant_id => @call_context.tenant_id,
+        :success => true, :created_at => Time.now, :updated_at => Time.now
+      )
+      results = plugin.search_payments(@kb_payment_id, 0, 100, [], @call_context).to_a
```

Comment:
@pierre Thanks for your time.

I did trigger the issue by a similar test instead of calling API from Kill Bill. Sorry for my incorrect usage.

I will modify my pr to fix `@response_model.search(search_key, context.tenant_id, offset, limit, :payment)` -> `@response_model.search(search_key, context.tenant_id, offset, limit)`